canonical-websites/www.ubuntu.com
# demo of orange hover state for homepage platform strip

## Done

- Added orange hover state for homepage platform strip icons and text

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that the icons and text on the platform strip change colour to orange on hover

## Issue / Card

Issue #1956
